integration-tests/tests/mocked/get-question/docker-compose.yaml

```diff
@@ -0,0 +1,15 @@
+version: "3.8"
+
+services:
+  step_function_local:
+    container_name: "getQuestion_stepfunction_local"
```

Comment:
Can you make the container the name more descriptive? e.g. `get-question-state-machine`
Since KBV has multiple step functions, we should be able to identify easier when doing `docker ps`